proto/strings.proto

```diff
@@ -3,19 +3,39 @@ syntax = "proto3";
 package strings;
 
 service StringsExtractor {
-   rpc ExtractStrings (StringsRequest) returns (StringsReply) {}
+	rpc ExtractStrings (StringsRequest) returns (StringsReply) {}
 }
 
 message StringsRequest {
-   // ID of file.
-   string id = 1;
+	// Binary ID.
+	string bin_id = 1;
 }
 
 message StringInfo {
+	// File offset to start of string.
 	uint64 location = 1;
+	// String in UTF-8 encoding.
 	string raw_string = 2;
+	// Size of string in bytes.
+	uint64 size = 3;
+	// Encoding of string.
+	Encoding encoding = 4;
```

Comment:
Interesting spin on this. The client gets all strings from all encodings, but can then filter (if they like) on a specific encoding. My initial thought was to add the encoding to the request instead, but this is better!